libs/domain-service/src/signers/index.ts

```diff
@@ -16,6 +17,11 @@ export const signDomainResolution = async (
   registryName: SupportedRegistries,
   challenge: string
 ): Promise<string | null> => {
+  if (!validateDomain(domain)) {
```

Comment:
This change should imply unit tests updates

Reply:
Test added but very redundant with the already existing tests for `validateDomain` itself.